src/js2xbox/events.cpp

```diff
@@ -60,18 +65,19 @@ int events::is_event_device(const struct dirent *dir) {
 }
 
 const char* find_existing_evdev() {
-    static const char *EVDEV_NAMES[] = {
-        "/dev/input/by-path/platform-odroidgo2-joypad-event-joystick",
-        "/dev/input/by-path/platform-odroidgo3-joypad-event-joystick",
-        "/dev/input/by-path/platform-singleadc-joypad-event-joystick"
-		
-    };
-
-    for (const char* dev : EVDEV_NAMES) {
-		if (access(dev, F_OK) == 0) {
-			return dev;
-		}
+	glob_t globbuf;
+
+	int rc = glob(EVDEV_WILDCARD, 0, NULL, &globbuf);
+	if (rc == 0) {
+		return *globbuf.gl_pathv;
```

Comment:
Don't know how this works in the context of releasing the memory. Probably doesn't matter in the grand scheme of things, or maybe we could copy the path buffer, release the glob, and let the caller take care of the path buffer copy? 🤔